inc/pagetop-btn/_scss/_pagetop-btn.scss

```
@@ -53,22 +66,72 @@
+	// このプラグインのフロント CSS には `.screen-reader-text` のグローバル定義が
+	// 無く、テーマ依存にもできない（当たらないとテキストが露出する）ため、
+	// `.page_top_btn` 配下にスコープして定義する。フォーカス時の露出は付けず
+	// （視覚はアイコン＋アウトラインが担う）、テキストは常に隠す。
+	.screen-reader-text {
+		position: absolute;
+		width: 1px;
+		height: 1px;
+		padding: 0;
+		margin: -1px;
+		overflow: hidden;
+		clip: rect(0, 0, 0, 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)